website_blog_excerpt_img/__manifest__.py

```diff
@@ -18,10 +19,8 @@
     "depends": [
         "website_blog",
         "html_image_url_extractor",
```

Comment:
@cristinamartinrod Would need add `oca_dependencies.txt` with `server-tools` to provide this dependency